analyzer/codechecker_analyzer/cmd/check.py

```diff
@@ -332,10 +332,11 @@ def add_arguments_to_parser(parser):
                                         "constraint solver.")
 
         analyzer_opts.add_argument('--z3-refutation',
-                                   action='store_true',
                                    dest='enable_z3_refutation',
-                                   default=argparse.SUPPRESS,
-                                   help="Enable the Z3 SMT Solver backend to "
+                                   choices=['on', 'off'],
+                                   default='on',
+                                   help="Switch on/off the Z3 SMT Solver "
+                                        "backend to "
```

Comment:
Please update the user guide too: https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#check-